tests/graceshutdown/graceshutdown_test.go

```diff
@@ -25,6 +38,7 @@ import (
 
 	_ "github.com/go-sql-driver/mysql"
```

Comment:
need to remove mysql reference?

Reply:
It's still being used at some places, for example, server/tidb_test. I'll remove them once tidb_test is fixed.

Of course, for go.mod inside  grace shutdown, it can be removed safely